### ISSUE TITLE
GPU fan will not go below 30%

### ISSUE THREAD

**Issue by JPL4494** [Rem0o/FanControl.Releases#2262]
**Describe the bug**

My Asus Phoenix 3060's fan will not drop below 30%. I am running version v179. I have tried putting the GPU in a friends computer, and the fan works like normal there, but I cannot get fan control to drop the speed below 30%. I tried following the tweaks from #2218, but I can't get the fans to drop to below 30%. I remember a previous version had it working, but somewhere along that version and the last few versions, the speed never drops down

**Is there a log.txt file next to FanControl.exe with recent date entries?**

No errors in the log file from the current version, here are the logs from an older version I upgraded from

```
11/21/2023 8:53:55 PM: System.ComponentModel.Win32Exception (225): An error occurred trying to start process 'C:\Users\JPL\Downloads\FanControl_net_7_0\Updater.exe' with working directory 'C:\Users\JPL\Downloads\FanControl_net_7_0\'. Operation did not complete successfully because the file contains a virus or potentially unwanted software.
   at System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   at FanControl.Domain.Utilities.ProcessUtilities.StartShellExecute(String path, String workingDir, String[] args)
   at FanControl.Domain.UpdaterService.UpdateVersion(UpdaterVersionInfo updateVersion, CancellationToken cancellationToken)
11/21/2023 9:23:25 PM: System.ComponentModel.Win32Exception (225): An error occurred trying to start process 'C:\Users\JPL\Downloads\FanControl_net_7_0\Updater.exe' with working directory 'C:\Users\JPL\Downloads\FanControl_net_7_0\'. Operation did not complete successfully because the file contains a virus or potentially unwanted software.
   at System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   at FanControl.Domain.Utilities.ProcessUtilities.StartShellExecute(String path, String workingDir, String[] args)
   at FanControl.Domain.UpdaterService.UpdateVersion(UpdaterVersionInfo updateVersion, CancellationToken cancellationToken)
11/21/2023 9:24:47 PM: NvAPIWrapper.Native.Exceptions.NVIDIAApiException: NVAPI_NVIDIA_DEVICE_NOT_FOUND
   at NvAPIWrapper.Native.GPUApi.GetThermalSettings(PhysicalGPUHandle physicalGPUHandle, ThermalSettingsTarget sensorTarget)
   at NvAPIWrapper.GPU.GPUThermalInformation.get_ThermalSensors()
   at FanControl.Domain.BackendProviders.NVApiWrapper.NVApiTempSensor.Update()
   at FanControl.Domain.BackendProviders.NVApiWrapper.NVApiWrapperBackendProvider.Update()
11/21/2023 9:24:47 PM: NvAPI failed. A complete refresh is required.
11/21/2023 10:33:57 PM: System.IO.IOException: Operation did not complete successfully because the file contains a virus or potentially unwanted software. : 'C:\Users\JPL\Downloads\FanControl_net_7_0\Updater.exe'
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.FileStreamHelpers.ChooseStrategyCore(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.File.OpenRead(String path)
   at FanControl.Domain.UpdaterService.GetHashString(String filePath)
   at FanControl.Domain.UpdaterService.UpdateVersion(UpdaterVersionInfo updateVersion, CancellationToken cancellationToken)
11/21/2023 10:34:21 PM: System.IO.IOException: Operation did not complete successfully because the file contains a virus or potentially unwanted software. : 'C:\Users\JPL\Downloads\FanControl_net_7_0\Updater.exe'
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.FileStreamHelpers.ChooseStrategyCore(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.File.OpenRead(String path)
   at FanControl.Domain.UpdaterService.GetHashString(String filePath)
   at FanControl.Domain.UpdaterService.UpdateVersion(UpdaterVersionInfo updateVersion, CancellationToken cancellationToken)
```

**Relevant hardware specs and setup**

- Asus 3060 Phoenix V2
- Asrock B450 Gaming ITX
- AMD Ryzen 5700G

**Comment by Rem0o** [comment]
The Nvidia API limits the minimum % to be applied on GPUs to 30% on recent ones.

**Comment by JPL4494** [comment]
> The Nvidia API limits the minimum % to be applied on GPUs to 30% on recent ones.

So checking the 0% override won't help? Is there a way to have the default fan curve run then? On my friends computer, it seems to run below 30%, and they don't have this program

**Comment by Rem0o** [comment]
Well, the 0% override simple disables FanControl completly for the GPU when the fancurve for it hits 0% in the software, hence letting it act on its own, which if the conditions are satisfied, the card will go 0 RPM.

**Comment by hubertsng** [comment]
> > The Nvidia API limits the minimum % to be applied on GPUs to 30% on recent ones.
> 
> So checking the 0% override won't help? Is there a way to have the default fan curve run then? On my friends computer, it seems to run below 30%, and they don't have this program

Read the help box next to the hardware curve override. This isn't a <30% fan curve fix, it's a 0% fan speed fix. Try turning on manual control on the GPU and setting it to 0% to see if that works. If it does, then you just have to make your curve not start until after the temperature you want it to spin, in which that I believe it will go straight to 30% as the minimum. I have my curve set as a graph I manually put and if I put 0% at 20 C with a straight line to 30% at 50 C, my GPU will always run at 30% unless my sensor temperature hits below 20 C. If I instead change the graph to say 0% at 49 C to 30% at 50 C, my GPU fan won't kick on until 49.1 C, in which it will go straight to 30%. This is with a NVIDIA RTX 3070 FE.

If you can't get 0%, then double check your GPU supports silent/passive mode.

**Comment by JPL4494** [comment]
@hubertsng I'm pretty sure my GPU supports it. When I booted with an Ubuntu live image instead, it wasn't spinning until I ran a jank load test. I can try again and see, but I'm at the point of just uninstalling because it's just not working

**Comment by hubertsng** [comment]
Just make sure whatever curve that is controlling it is reading 0% in the "Curves" section or set it as just 0% in "Manual control". If the curve selected in the "Curves" section or the "manual control" is not returning 0%, the GPU will return a 30% (?) in the "Controls" portion.

**Comment by JPL4494** [comment]
I have the manual set to 15% and it won't drop that low. I tried to do the manual setting before and it didn't work, but I gave it another go and it still isn't dropping below 30%

![fan_manual_fail](https://github.com/Rem0o/FanControl.Releases/assets/23563578/96571f9e-1ca0-432d-92cd-967c3a019e5d)
![gpu_fan_curve](https://github.com/Rem0o/FanControl.Releases/assets/23563578/7d2715d7-fe68-4f01-a12a-c1926612cdee)

**Comment by Rem0o** [comment]
Do you have the override ticked on, and remove force apply.

**Comment by JPL4494** [comment]
I have tried every combo, and no dice. I just tried again with the override on and the force off, and it still won't drop below 30%

**Comment by Rem0o** [comment]
Is the card set with an OC, high power mode or something like that in the nvidia control panel?

**Comment by JPL4494** [comment]
@Rem0o I don't see that option anywhere on the control panel. I tried looking it up online, and it seems like that is for laptops, not desktops?
![nvidia_panel](https://github.com/Rem0o/FanControl.Releases/assets/23563578/13732493-6846-47f8-b62a-d6e6b95edde1)

**Comment by JPL4494** [comment]
Nope, it is set to normal and balanced profiles in those spots, and the fan ramped up once I started the software to 30%. I guess it won't work for me 😭

**Comment by Rem0o** [comment]
Screenshot your sensor configuration. Post it here.

Then screenshot your fan control home page with everything visible and the 30 % problem happening.

**Comment by Calancea-Gabriel** [comment]
> Nope, it is set to normal and balanced profiles in those spots, and the fan ramped up once I started the software to 30%. I guess it won't work for me 😭

I have your same problem, mine won't go under 29% even with manual control turned to 0

**Comment by JPL4494** [comment]
@Rem0o sorry for the delay, life popped up for a bit. I've tweaked my profile a lot from the start of this ticket, so IDK if I might've finger fumbled on something since then. Let me know if you need more screenshots
![sensors](https://github.com/Rem0o/FanControl.Releases/assets/23563578/95a87f27-00b4-4ab8-9b7c-ab84c82a7d31)
![fancontrol-dashboard2](https://github.com/Rem0o/FanControl.Releases/assets/23563578/44a008fb-4fab-4e83-9255-c9b4f2f595c7)

**Comment by Rem0o** [comment]
@JPL4494 Your fan curve ask for 6.3%, which is not 0.  You can't use the [1-29] range, it must be either 0, or 30 and above.  You might want to use a graph for this, much easier to understand.

 Also, remove "force apply" from the Nvidia Control card.

**Comment by rajjejosefsson** [comment]
Having similar issue as well, what do we miss?
 
![FanControl_nijTmpyMSs](https://github.com/Rem0o/FanControl.Releases/assets/13612444/c7faef96-07da-4c6b-9e8b-12f3c1768cf9)

![vJNZYF2aMU](https://github.com/Rem0o/FanControl.Releases/assets/13612444/92928e28-7197-4e87-9c56-60adcf492274)

**Comment by Rem0o** [comment]
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/801101ec-ef5d-4e52-91cc-e1c4e291ac52)

**Comment by hubertsng** [comment]
> Having similar issue as well, what do we miss?
> 
> ![FanControl_nijTmpyMSs](https://private-user-images.githubusercontent.com/13612444/301666085-c7faef96-07da-4c6b-9e8b-12f3c1768cf9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4OTYxMDEsIm5iZiI6MTcwNjg5NTgwMSwicGF0aCI6Ii8xMzYxMjQ0NC8zMDE2NjYwODUtYzdmYWVmOTYtMDdkYS00YzZiLTllOGItMTJmM2MxNzY4Y2Y5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjAyVDE3NDMyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ1NDljYWYxODU4NDA4MDA3ZWJhNjljODg2ZjU3M2UyZTQyODRmMTU4NWI0ZDJkMDFmNjliMGQ0MmNlNDk5OGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.D09bDg-LeVEDvlcYcr-kjID3BItkaC5UCJ_Cjt7DoYM)
> 
> ![vJNZYF2aMU](https://private-user-images.githubusercontent.com/13612444/301666289-92928e28-7197-4e87-9c56-60adcf492274.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4OTYxMDEsIm5iZiI6MTcwNjg5NTgwMSwicGF0aCI6Ii8xMzYxMjQ0NC8zMDE2NjYyODktOTI5MjhlMjgtNzE5Ny00ZTg3LTljNTYtNjBhZGNmNDkyMjc0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjAyVDE3NDMyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNkNTM1NDllOWVmZTkyMDU5ZDVjMWM3OTFlMWJlZGNmNGY3NGQ4OTJmZjMwYmUwYjIxMjdiMGVkNDY5NzNiYjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.T96qcjTeA5TB2ZbrQRuLpNSHjhi9x0deJ_Pl3IiKcno)

Guys, make the actual fan curve that is controlling the GPU set to 0% if you're going to send a screenshot. Your curve, "Max of Graphs", isn't reporting 0% fan speed. That's the thing of interest since that's the thing that is controlling the GPU fan speed and not the GPU temp. 

![image](https://github.com/Rem0o/FanControl.Releases/assets/16064810/6660a8b8-8697-4dbf-9046-2a8a71f2a7ed)
![image](https://github.com/Rem0o/FanControl.Releases/assets/16064810/bdeea768-82a2-44d6-b85c-424c88125ae3)

**Comment by JPL4494** [comment]
@Rem0o @hubertsng I created a graph curve, and I am still seeing the speed pinned when it should be 0% from the graph. If I have the 0% override enabled, the speed is 40%, but when I disable it the speed is 30%. I disabled the force apply and it still hasn't changed

**Comment by Rem0o** [comment]
Shoot your graph. 

It should look like this:
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/4f2c2b8c-6f64-4b7c-ab34-695a7a34759e)

**Comment by JPL4494** [comment]
@Rem0o Whoops, looks like I didn't drag the graph into my comment, sorry! Here's what I have, I tweaked it to how yours looks and it still stays at 30%. If I can get it working, I can massage the curve to be better suited for my case and room

As for temps, it's below 22 degrees C
![0-percent-fan-curve](https://github.com/Rem0o/FanControl.Releases/assets/23563578/1bf9041a-77fb-4754-b709-e44288aa4d28)
![0-percent-fan-curve-2](https://github.com/Rem0o/FanControl.Releases/assets/23563578/6164e0a2-e022-4ec3-8e7b-cfe43fdc5cfb)

**Comment by Rem0o** [comment]
If you exit fan control or disable your GPU controls, does it actually stops instantly at those temperatures?  Because that's what we need in the first place, otherwise we are chasing nothing.

**Comment by hubertsng** [comment]
What happens if you use other fan control settings for your gpu? If it's foss, you can probably see how it hooks into controlling your gpu fan speed. Maybe look at your gpu manufacturer's software.

**Comment by eproxus** [comment]
I'm having the same problem. It seems that even though the graph is at 0%, the graphics card is overridden to 30% (as seen by the question mark). _Sometimes_ it drops to 0%, but it is not reliable, and it seems once it has passed 30% it seldom drops down to less than 30% again. I haven't been able to figure out a pattern of when exactly it does this.

![image](https://github.com/Rem0o/FanControl.Releases/assets/112878/2b51b1fe-d27c-436c-9d50-1287a9cab1fe)

**Comment by Rem0o** [comment]
@eproxus  do you got this enabled?
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/62498b6f-c547-456a-b0cd-d9bad342c758)

Then you need to figure out the conditions your card will go 0 RPM by itself, and play around them.  You won't ever have complete control of that behavior.

**Comment by eproxus** [comment]
@Rem0o Yes, I do. As you explained before (and I have now understood) you are correct in that it works as intended.

For anyone needing a clarification: The card itself has a temperature where goes to 0% fan speed (for my RTX 4070 FE it is below 35°C). This has to be taken into account with the fan speed control graph (or whatever control you use), otherwise you may end up creating an "impossible" scenario. The goal here is to figure out at what temp your card goes to 0%, and make the graph align with that (set the percentage to 0% below that temp, to whatever you want above that temp). Once your card has started spinning its fans, _it will not go to 0% if you tell it to, **only when it is below its trigger temperature**_.

**Comment by Rem0o** [comment]
@eproxus  couldn't have said it better. Might steal that explanation actually 😅

**Comment by eproxus** [comment]
@Rem0o Please do! ❤️

**Comment by Rem0o** [comment]
Created a wiki page: https://github.com/Rem0o/FanControl.Releases/wiki/Nvidia-30%25-and-0-RPM

**Comment by JPL4494** [comment]
It would seem Asus, in their infinite wisdom, removed the 0% mode in a firmware update. I don't want to downgrade because it would reintroduce the other issues, so I guess I'm stuck 😿

**Comment by medapayne** [comment]
> @Rem0oOui je le fais. Comme vous l'avez expliqué précédemment (et j'ai maintenant compris), vous avez raison dans la mesure où cela fonctionne comme prévu.
> 
> Pour tous ceux qui ont besoin d'une précision : la carte elle-même a une température qui va jusqu'à 0% de vitesse du ventilateur (pour ma RTX 4070 FE, elle est inférieure à 35°C). Ceci doit être pris en compte avec le graphique de contrôle de la vitesse du ventilateur (ou tout autre contrôle que vous utilisez), sinon vous risquez de créer un scénario « impossible ». Le but ici est de déterminer à quelle température votre carte passe à 0 % et d'aligner le graphique sur cela (réglez le pourcentage à 0 % en dessous de cette température, à ce que vous voulez au-dessus de cette température). Une fois que votre carte a commencé à faire tourner ses ventilateurs, _elle n'atteindra pas 0% si vous le lui dites, **uniquement lorsqu'elle sera en dessous de sa température de déclenchement**_ .

Hello 

sorry with my school English I understand the settings @eproxus  could you show me in screen of your configuration please.
I have a 3070 FE and I can't get the fans to stop below 40 degrees.
Thanks in advance

**Comment by Rem0o** [comment]
@medapayne  cette limitation de 40 degrés est intégré dans la carte graphique. FanControl ne peut changer cette limitation. Vous devez construire votre profile de ventilation autour de cette limitation, c'est à dire en assumant que les fans ne peuvent arrêter si la carte n'est pas en dessous de 40 degrés.

**Comment by medapayne** [comment]
> @medapayne cette limitation de 40 degrés est intégré dans la carte graphique. FanControl ne peut changer cette limitation. Vous devez construire votre profile de ventilation autour de cette limitation, c'est à dire en assumant que les fans ne peuvent arrêter si la carte n'est pas en dessous de 40 degrés.

Thank you for your reply,
This is exactly my whole problem
As I write to you, my graphics card displays a temperature of 36° and the fans are running at 1000RPM
While sometimes it will be at 45 and the fans will be at 0 RPM
I can't understand this damn logic which means that sometimes it works and sometimes it doesn't.
What I would like is that below 40° the fans are stopped and as soon as we reach 41° they start up.
I agree about creating a curve, that's exactly what I want. I want to do it but at the moment I can't understand how you did it, I'm totally lost

**Comment by limetto** [comment]
So I think the problem is getting the points where it starts spinning and where it turns of again right.
My RTX3700 starts at 63° and stops again at 32°
So i configured this:
![fan1](https://github.com/Rem0o/FanControl.Releases/assets/17469894/9646959f-3518-438a-8eb9-a1d1d880562c)
![fan2](https://github.com/Rem0o/FanControl.Releases/assets/17469894/691e5849-5ca8-4297-b2c5-6538b5d1d384)
I get 0% heating up until 63° and flat 30 for idle cooldown from 63° down to 32°, once it hits 32° it spins down and is idle again.

**Comment by medapayne** [comment]
> So I think the problem is getting the points where it starts spinning and where it turns of again right. My RTX3700 starts at 63° and stops again at 32° So i configured this: ![fan1](https://private-user-images.githubusercontent.com/17469894/313388919-9646959f-3518-438a-8eb9-a1d1d880562c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA1ODA4MDYsIm5iZiI6MTcxMDU4MDUwNiwicGF0aCI6Ii8xNzQ2OTg5NC8zMTMzODg5MTktOTY0Njk1OWYtMzUxOC00MzhhLThlYjktYTFkMWQ4ODA1NjJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE2VDA5MTUwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI3YWQwODBlMDg3NDkzN2FhNTY4MmEyOTM3M2JhZjhiNDNjNGY3NjAwMDQ2MzBiYmQ5OGYzNzQ3MzZlNWIwZTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._My0V3yJ_oWG9pF9j0nJAw-T-QcxYYJ2eqrHiNHzCfI) ![fan2](https://private-user-images.githubusercontent.com/17469894/313388925-691e5849-5ca8-4297-b2c5-6538b5d1d384.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA1ODA4MDYsIm5iZiI6MTcxMDU4MDUwNiwicGF0aCI6Ii8xNzQ2OTg5NC8zMTMzODg5MjUtNjkxZTU4NDktNWNhOC00Mjk3LWIyYzUtNjUzOGI1ZDFkMzg0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE2VDA5MTUwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNiY2VjNDZhNmE2NTc1NTA0NDE1NjYyNWZlZDNkMDNkOGFlYWRkMTE5ZWRhM2U1MWNmMDM2YjI5YmVmNGY5MjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Blog32gxT5jDFUSVPfH5ZeAOHM_0h4RDFi2GJvGXXWM) I get 0% heating up until 63° and flat 30 for idle cooldown from 63° down to 32°, once it hits 32° it spins down and is idle again.

Hello @limetto 

Thank’s for your reply

To be sure, does this work for you? Or what you show in your screenshots and to mean that it doesn't work?
I ask this because if it works for you I will rely on your screenshots to try to create my fans curve.

**Comment by limetto** [comment]
Works fine for me after tinkering around with it for a long time.

**Comment by medapayne** [comment]
> Fonctionne bien pour moi après avoir bricolé pendant longtemps.

Great new and thx 
sorry i have some questions :

- you have RTX 3070 , wright?
- why don't use a graph like that 

![image](https://github.com/Rem0o/FanControl.Releases/assets/25185208/b4e4ae38-c205-443b-9347-1b2a324ce311)
 
- in my case ( 3070 FE ) i conclude idle mode ( 0% ) start when my temperature is under 35 degrees and wattage is under 21W

**Comment by nonamejuju** [comment]
Please note that from my side the fans never go to 0% when I have 2 monitors pluged-in (32W of power) but do when 1 using only 1 monitor (less than 20W) even if my temp are below 34°C. I haven't find a solution to this yet.

**Comment by Kjeld02** [comment]
Hey @limetto,

I tried replicating your settings, but I'm encountering an issue. Whenever my GPU surpasses 37 degrees, the fan kicks in and stays on, even when the temperature drops below 32 degrees. Did you face a similar problem?

**Comment by limetto** [comment]
> Hey @limetto,
> 
> I tried replicating your settings, but I'm encountering an issue. Whenever my GPU surpasses 37 degrees, the fan kicks in and stays on, even when the temperature drops below 32 degrees. Did you face a similar problem?

No, mine stays quite until 63°. Maybe you model handles spinup under high load differently.

**Comment by Sebo1090** [comment]
In my case with INNO3D GeForce RTX 4070 Ti X3 card I have no real impact on the operation of its fans, I tested the author's program and msi afterburner, well, no matter what values I set, the factory software of the card automatically sets the minimum speed of the fans at 30% and 100% when reaching 80 degrees, no combinations in the settings do not help, what's funny when I turn off any program, the card turns on “sleep” mode and at the minimum temperature the fans stand still, in my case the lower speed limit is not a problem, at 30% I can't hear the fans anyway, the problem is when the card reaches 80 degrees, then the card's software automatically sets their speed to 100%, no matter what program I have active to control them, this is extremely annoying, because then the card begins to resemble a jet when it comes to its noise, what is worse, as soon as the temperature drops to 78 degrees, the fans reduce their speed, 
and when the temperature rises again, the fans turn on jet mode.... and so on and so forth,...it goes crazy, well in my case in the next card I will pay more attention to its cooling system

**Comment by wer1976** [comment]
A late comment. It seems like the magic 30% is at least on my card (MSI Ventus 4070 super OC) always 30% of the frequency/RPM/speed of the ORIGINAL fans. On my card, 30% of the orignal fans is (approx) 1300 RPM. So, if you deshroud your GPU and use custom fans, the GPU's fan controller tries to run those custom fans at 1300 RPM. This is not the usual behaviour of fans where 30% corresponds to the PWM duty cycle which should make the fans rotate at 30% of their max speed.

I tested this by using various fans and fan control. 30% always results in 1300 RPM regardless of the fan (unless the fan had a lower max, then it would run at that max...)

**Comment by mrsieb** [comment]
> [@Rem0o](https://github.com/Rem0o) Yes, I do. As you explained before (and I have now understood) you are correct in that it works as intended.
> 
> For anyone needing a clarification: The card itself has a temperature where goes to 0% fan speed (for my RTX 4070 FE it is below 35°C). This has to be taken into account with the fan speed control graph (or whatever control you use), otherwise you may end up creating an "impossible" scenario. The goal here is to figure out at what temp your card goes to 0%, and make the graph align with that (set the percentage to 0% below that temp, to whatever you want above that temp). Once your card has started spinning its fans, _it will not go to 0% if you tell it to, **only when it is below its trigger temperature**_.

Well i figured out that's not exact the case, I mean the Temperature goes down in this circumstance but the related values is how much Power the card is using.  And this is related to GPU Clock and VRAM Clock. My 4090 goes to 0 RPM with 42° degrees as well. Power have to be below 50W.  (at least on the 4090) for this GPU Clock  and VRAM Clock have to be under 800Mhz. Afterwards the card cools down as well.  

Two things i struggle a lot was different Hz on different Monitors one on 144Hz one on 120Hz. since both are running with 120 no issues any more even on 4k  both on 144Hz as well. If the refresh rate is different on the screen the VRAM Glock goes not under 10kMhz.  And last but not least i had to remove all Nvidia Drivers with DUU Tool from Guru 3D and reinstall fresh. (Don't know which setting was preventing the mode)  Any way this is a nice IDLE State:

![Image](https://github.com/user-attachments/assets/ba1b6257-0415-4c3d-ac0c-fce19a13288b)

**Comment by eproxus** [comment]
As I just discovered when switching to a new 4K monitor: The card might just draw more power under certain scenarios (desktop refresh rate, number of monitors, resolution of said monitors etc.) which _will_ prevent it from going below the internal 0% threshold of the card. Just because you configured a FanControl threshold where you want it to go to 0% doesn't mean the card has to agree with you.

In other words: you can only _guide_ the fan percentage so much, if something makes the card draw more power you might not go below the internal threshold temperature all the time.

For example, with my setup (4K 240hz monitor plus RTX 4070 FE) the card often goes to 0% but not always, depending on what is going on on the desktop (multiple windows, videos playing etc.).

**Comment by Jlpeaks** [comment]
Zotac 5070ti owner here who thought I had a gremlin in my machine ( I did a full windows reinstall and everything to make sure no software was messing with fans).

The long story short is that I thought I had an issue as my fans were spinning up well ahead of what I had asked them to on my fan curve.. as low as 32°. But this would only happen under a moderate workload, In my case Fallout 4 maxed out at 165 frames triggers it but the same game capped to 60 doesn’t.

I then started dreading that my week old card didn’t have its cooler seated properly or that the Vbios had somehow glitched out despite me not doing anything to it.

Turns out it’s just because the ‘fan stop’ curve on the bios also takes power draw as a variable.
I managed to find this buried in a support forum from a Zotac rep, speaking about a 4080;

“The fan stop thresholds are below
GPU temperature >= 55C OR GPU loading >= 120W, than fan ON
GPU temperature < 55C AND GPU loading < 120W, than fan OFF”

Bit of relief as at least my card is behaving like intended. Now it’s just a question of if those intentions are right.. surely with such low power draw limits on such high TDP cards, the fan stop feature is useless outside of idle scenarios. I’ve looked into it and Asus also have this power draw threshold but I can’t find anything from other manufacturers.

I think I get why they have it built in, they don’t want your card to frequently go from low 30’s idle to 50’s under load. That variance will deteriorate the materials but perhaps they should have the power threshold have an upper limit on its duration. For example if my card has been pulling 150w for multiple minutes and remains sub 50, it’s probably fine. I’m a bit disappointed with this card as a result as one of the features (Freeze/fan stop) is pretty useless.

**Comment by thehaplo** [comment]
Has anyone figured it out if this "problem" is hard baked into GPU? Because whatever I do, my ASUS RTX 2060 6G just doesn't want to drop under 800 RPM. Even tho it shows 24% and 36% respectively, all three fans just sway between 797~802 RPM. I have no other fan controller software. 

Also same with my Deep Cool AK620 CPU cooler. Even if I take the manual control and drop to 0%, this time even if it show 0%, it just won't drop under 560~ RPM.

**Comment by ssjrocks** [comment]
i have an asus 5080 tuf and i couldnt get fans to stop, read about the nvidia override in settings and turned it on... fans were stopping and starting as required for like a day or 2 , i dont know if it was an update or if something changed on my end but im back to the ? next to the 30% min when it tries to turn them off regardless of the nvidia override.. have balanced power profile set... and fan control is the only software controlling any fans on my setup
